### PR TITLE
Wms links with https

### PR DESCRIPTION
**Pull request by pfanguin**

### CODE REVIEW

**Comment by oterral** [comment]
@pfanguin can you squash the 2 commit pls

**Comment by pfanguin** [comment]
jenkins test

**Comment by oterral** [comment]
jenkins test

**Comment by oterral** [comment]
jenkins test

File: src/js/ImportWmsController.js [hunk 1]
```diff
@@ -39,13 +38,13 @@
...
-           'http://www.sogis1.so.ch/cgi-bin/sogis/sogis_grundbuch.wms?',
+           'http://www.sogis1.so.ch/wms/grundbuchplan?',
+           'http://www.sogis1.so.ch/cgi-bin/sogis/sogis_orthofoto.wms?', 
            'http://www.sogis1.so.ch/cgi-bin/sogis/sogis_bpav.wms?',
            'http://www.sogis1.so.ch/cgi-bin/sogis/sogis_strassenkarte.wms?',
-           'http://www.sogis1.so.ch/cgi-bin/sogis/sogis_ortho.wms?',
            'http://www.sogis1.so.ch/cgi-bin/sogis/sogis_dtm_dom.wms?',
-           'https://secure.asitvd.ch/proxy/ogc/vd-wms?',
-           'http://wms.geo.bs.ch/wmsBS?',
+           'http://secure.asitvd.ch/proxy/ogc/vd-wms?',
```
Comment (oterral) [inline src/js/ImportWmsController.js]: Why removing the https ?

**Comment by oterral** [comment]
According to @davidoesch  mail:

 - Wmss are  missing  see list  -> http://geo.gr.ch/index.php?id=113

**Comment by oterral** [comment]
Thx ofr this

**Pull request merged**